tuxemon/core/player.py

```diff
@@ -49,3 +49,28 @@ def __init__(self, npc_slug):
 
         # Game variables for use with events
         self.game_variables = {}
+
+        # Number of steps
+        self.game_variables['steps'] = 0
+
+    def move(self, time_passed_seconds):

```

Comment:
this method is called at least once per a frame, and is responsible for any and and all movements from the player and from external events.  this means that if an external action moves the player additional "steps" are created.  since this is not a huge feature yet, leaving the logic here is fine, but please add this comment:
```
TODO: this will also record involuntary steps.  refactor so that only movements from the player are recorded.
```

Reply:
I'll add this comment, but are external actions uses the npc.path to move the player ? I guess it does, but if it's not the number of steps could be incremented when the player appends his path.

Reply:
internally, all movements use path.  even pressing a direction key will create a path to the next tile.  external actions (typically pathfind) creates a path.  at this time, there is no way to know what created the path...an external action or a player key, making this ticket complicated.  so, im ok with making it a `TODO` for now.